services/graph/pkg/unifiedrole/unifiedrole.go

```diff
@@ -0,0 +1,346 @@
+package unifiedrole
```

Comment:
What reason to copy the mapping from a reva?

Reply:
We're basically moving it here. See: https://github.com/cs3org/reva/pull/4336